cczoo/cross_lang_framework/docker/start_clf_client_container.sh

```diff
@@ -39,9 +39,13 @@ docker run -it \
     --device=/dev/sgx_enclave:/dev/sgx/enclave \
     --device=/dev/sgx_provision:/dev/sgx/provision \
     --add-host=pccs.service.com:${ip_addr} \
+    --add-host=`hostname`:${ip_addr} \
     -e no_proxy=${no_proxy} \
     -e http_proxy=${proxy_server} \
     -e https_proxy=${proxy_server} \
     -v /home:/home/host-home \
+    -v `pwd -P`/../tools/ca_cert.crt:/clf/cczoo/cross_lang_framework/clf_client/app/certs/ca_cert.crt \
```

Comment:
please mention that if certification is not ready, user should run the script in tools folder to generate certification first